providers/microsoft/mssql/pyproject.toml

```diff
@@ -58,7 +59,8 @@ requires-python = "~=3.10"
 dependencies = [
     "apache-airflow>=2.10.0",
     "apache-airflow-providers-common-sql>=1.23.0",
-    "pymssql>=2.3.5",
+    "pymssql>=2.3.4",
```

Comment:
Oh, you even accept a downgrade for `pymssql` :-D

Reply:
😱